### PR TITLE
Add link

### PR DESCRIPTION
**Pull request by Xxy1016**
Changelog
---------

### Added

- Added html link

### Security

Checklist
---------

- [x] Test
- [ ] Self-review
- [ ] Documentation

### CODE REVIEW

File: src/components/Home/Home.test.tsx [hunk 1]
```diff
@@ -11,3 +12,6 @@ test("renders app without error", () => {
     </BrowserRouter>
   );
 });
+
+const linkToTest = screen.getByRole("link", { name: /link to test/i });
+expect(linkToTest.getAttribute("href")).toMatchInlineSnapshot();
```
Comment (QubitPi) [inline src/components/Home/Home.test.tsx:17]: 👍🏻 for using `screen.getByRole("link", { name: /link to test/i });`, **very brilliant**!

Adding links to "GitHub" and "Help" is a very important feature and, therefore, deserves its own unit test. A unit test in React (or JavaScript in general) is described by a [`test` macro](https://jestjs.io/docs/getting-started) and we need to test both "GitHub" and "Help" links.

I saw you also have **snapshot testing** there (`toMatchInlineSnapshot`) and this is indeed very nice. Snapshot testing though does have it's downside in some cases. _In general, Messier-61 do not use snapshot testing_

Putting everything together, I recommend enhancing the code to become something like the following:

```typescript
test("GitHub and Help buttons point to the expected locations", () => {
  render(
    <BrowserRouter>
      <Home />
    </BrowserRouter>
  );

  const githubLink = screen.getByRole("link", { name: /GitHub/i });
  expect(githubLink.getAttribute("href")).toBe('https://github.com/paion-data/Messier-61');

  const helpLink = screen.getByRole("link", { name: /Help/i });
  expect(helpLink.getAttribute("href")).toBe('https://paion-data.github.io/Messier-61/');
});
```

**Review by QubitPi — CHANGES_REQUESTED** [review]
I'm very happy that you added unit tests to this PR! 💯 Just some polishing left there and we are ready to merge this PR.

File: src/components/Home/Home.css [hunk 2]
```diff
@@ -97,6 +107,7 @@ main {
   top: 5%;
   right: 5%;
 }
+
```
[the remark this answers is not in the document]
Comment (Xxy1016) [reply src/components/Home/Home.css:110]: 大概是因为我格式化了一下代码 所以自动添加了一行空格

File: src/components/Home/Home.css [hunk 3]
```diff
@@ -4,12 +4,17 @@
   box-sizing: border-box;
 }
 
+a {
+  text-decoration: none;
```
[the remark this answers is not in the document]
Comment (Xxy1016) [reply src/components/Home/Home.css:8]: 因为HTML里的超链接会自动带有下划线样式 所以在CSS中取消了下划线

[the remark this answers is not in the document]
Comment (QubitPi) [reply src/components/Home/Home.css:8]: Ok

**Review by QubitPi — APPROVED** [review]

**Pull request merged**